libs/helm/package.json

```diff
@@ -3,13 +3,13 @@
 	"version": "0.0.1-alpha.550",
 	"sideEffects": false,
 	"peerDependencies": {
-		"@angular/cdk": "^19.0.0",
-		"@angular/common": "^19.0.0",
-		"@angular/core": "^19.0.0",
-		"@angular/forms": "^19.0.0",
-		"@angular/router": "^19.0.0",
-		"@ng-icons/core": "^31.4.0",
-		"@ng-icons/lucide": "^31.4.0",
```

Comment:
Could we also relax the dependencies here too, so we support multiple major versions